docs/fundamentals/code-analysis/quality-rules/ca1068.md

```diff
@@ -24,6 +24,8 @@ ms.author: mavasani
 
 A method has a <xref:System.Threading.CancellationToken> parameter that is not the last parameter.
 
+By default, this rule analyzes the entire codebase, but this is [configurable](#excluded-symbols).
```

Comment:
```suggestion
By default, this rule analyzes the entire codebase, but this is [configurable](#configurability).
```

Reply:
Other configurability might not be related to the configuration of which part of the codebase is analyzed.

Reply:
I wanted to suggest a different arrangement of the configuration section, but I couldn't create a PR to your fork, so I had to commit directly to your branch instead.